pages.es/common/amass-intel.md

```diff
@@ -0,0 +1,28 @@
+# amass intel
+
+> Recopila información de código abierto sobre una organización, como dominios raíz y ASNs.
+> Más información: <https://github.com/OWASP/Amass/blob/master/doc/user_guide.md#the-intel-subcommand>.
+
+- Busca dominios raíz en un rango de direcciones IP:
+
+`amass intel -addr {{192.168.0.1-254}}`
+
+- Utiliza métodos activos de reconocimiento:
+
+`amass intel -active -addr {{192.168.0.1-254}}`
+
+- Busca dominios raíz relacionados con un dominio:
+
+`amass intel -whois -d {{nombre_de_dominio}}`
+
+- Busca ASNs pertenecientes a una organización:
+
+`amass intel -org {{nombre_de_organizacion}}`
+
+- Busca dominios raíz pertenecientes a un determinado número de sistemas autónomos:
+
+`amass intel -asn {{asn}}`
```

Comment:
```suggestion
- Encuentre dominios raíz pertenecientes a un Número de Sistema Autónomo específico:

`amass intel -asn {{string}}`
```

Almost the same issue as above.
Optional fix: `asn` term duplication can be reduced.
Warning: failed to deduce value type of `-asn` option, `string` used as a fallback.